### DISCUSSION TITLE
multiuser access

### DISCUSSION THREAD

**Discussion by dlannom** [lwindolf/liferea#1042] [General]
I'm thinking about setting up  [gaucamole](https://guacamole.apache.org/) to access liferea from multiple computers in my house.   Is it safe to allow simultaneous access, or should I restrict it to one at a time?

Thanks,
Dan

**Comment by lwindolf** [comment]
As gaucamole is a remote desktop tooling accessing only a single Liferea instance it should be perfectly fine.
